### PR TITLE
Update the label filter for when scope is 'hmda'

### PR DESCRIPTION
**Pull request by LinuxBozo**
Noticed for edits like S025, when the scope is `hmda` that the label filter can't look up the labels from the file spec properly because the `hmda` scope doesn't exist in the implementation, nor in the file spec itself. So reset scope based on `lineNumber`.

### CODE REVIEW

File: app/partials/errorDetail.html [hunk 1]
```diff
@@ -14,8 +14,8 @@
     </thead>
     <tbody ng-repeat="item in error.errors | paginate:start():end()">
         <tr ng-repeat="(key, value) in item.properties">
-            <th class="id" scope="row" rowspan="{{item.properties | keyLength}}" ng-if="$first">{{item.loanNumber || item.lineNumber}}</th>
-            <td class="description">{{key | hmdaLabel:error.scope}}</td>
+            <td class="id" scope="row" rowspan="{{item.properties | keyLength}}" ng-if="$first">{{item.loanNumber || item.lineNumber}}</td>
```
Comment (poorgeek) [inline app/partials/errorDetail.html]: Why did you change the `th` to a `td`? Shouldn't the first column be designated as a row header?

Comment (LinuxBozo) [inline app/partials/errorDetail.html]: Not sure how that got changed.. yes, it should be `th`. Must have been something Sublime did for me and I didn't notice. Good catch.
